src/main/java/life/qbic/omero/BasicOMEROClient.java

```diff
@@ -247,7 +247,7 @@ public String getImageDownloadLink(long imageID) {
       ImageData image = browse.getImage(this.ctx, imageID);
       disconnect();
       if (image.getFormat() != null) {
-        res = hostname + "/omero/webgateway/archived_files/download/" + imageID + "/";
+        res = hostname + "/omero/webgateway/archived_files/download/" + imageID + "?server=1&bsession=" + this.sessionId;
```

Comment:
NIT: a more declarative name is be favored

Reply:
instead of result you mean? Will change that, good point.

Reply:
b50b3e3